pom.xml

```diff
@@ -45,10 +45,10 @@
         <auto-matter.version>0.15.3</auto-matter.version>
         <auto-value.version>1.7.4</auto-value.version>
         <logback.version>1.2.3</logback.version>
-        <jackson.version>2.11.2</jackson.version>
+        <jackson.version>2.12.3</jackson.version>
```

Comment:
In the spotify internal java/bom we currently have 2.12.2, but they should be compatible I guess.